src/bluetoothctl.py

```diff
@@ -0,0 +1,204 @@
+# ReachView code is placed under the GPL license.
```

Comment:
Don't add this file yet (to git). I'm still figuring out how we're going to use this and I already have a branch that adds it (I just haven't opened a pull request yet). Go ahead and take a look at that branch. Ignore the leader.py changes, they shouldn't be there, but take a look at how I'm using it.

It also shouldn't be in this PR. Adding it to what we have right now should be a separate pull request because main.py currently does not work for the leader.